deker/ABC/base_array.py

```diff
@@ -392,26 +392,54 @@ def update_custom_attributes(self, attributes: dict) -> None:
         self.logger.info(f"{self!s} custom attributes updated: {attributes}")
 
     def _create_meta(self) -> str:
-        """Serialize array into metadata string."""
+        """Serialize array into metadata JSON string."""
+
+        def serialize_value(
+            val: Any,
+        ) -> Union[Tuple[str, int, float, tuple], str, int, float, tuple]:
+            """Jsonify attribute value.
+
+            :param val: complex number
+            """
+            if isinstance(val, datetime):
+                return val.isoformat()
+            if isinstance(val, np.ndarray):
+                return val.tolist()  # type: ignore[attr-defined]
+            if isinstance(val, complex):
+                return str(val)
```

Comment:
On deserialization how you gonna distinct complex"2 + 4i" from string "2 + 4i" ? As I can see, there is no type conversion on  (check `_create_from_meta` method)

Reply:
It's an interesting question. I was ready to catch serialized complex numbers with a regexp.
But according to this [test case](https://github.com/openweathermap/deker/blob/fix/bugs_in_attributes/tests/test_cases/test_schemas/test_attributes_schema.py#L176-L269) they a deserialized without any problems.

Magic)